config/samples/placement_v1beta1_placementapi.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: placement.openstack.org/v1beta1
 kind: PlacementAPI
-namespace: openstack
 metadata:
   name: placement
+  namespace: openstack
```

Comment:
Hmm, it looks like actually setting the `namespace` here causes problems for the KUTTL tests:

```
logger.go:42: 14:34:07 | placement_scale/2-patch_placement_deploy | running command: [sh -c oc patch placementapi -n $NAMESPACE placement --type='json' -p='[{"op": "replace", "path": "/spec/secret", "value":"osp-secret"}]'
        ]
    logger.go:42: 14:34:07 | placement_scale/2-patch_placement_deploy | Error from server (NotFound): placementapis.placement.openstack.org "placement" not found
```

I think because the sample gets created in the `openstack` namespace but the test is using `placement-kuttl-tests` for the `$NAMESPACE`.

Reply:
It might be better to just drop the `namespace` completely, as it seems that is what we do elsewhere for the same sort of samples (example [1]) that are pulled into KUTTL tests.

[1] https://github.com/openstack-k8s-operators/keystone-operator/blob/main/config/samples/keystone_v1beta1_keystoneapi.yaml

Reply:
I can do that, but its strange, because kustomize should still set it to the correct namespace.

Reply:
In this KUTTL case, I don't think `kustomize` is involved.  It just directly applies the CR [1] [2]. 

[1] https://github.com/openstack-k8s-operators/placement-operator/blob/main/tests/kuttl/tests/placement_scale/01-deploy_placement.yaml
[2] https://github.com/openstack-k8s-operators/placement-operator/blob/main/tests/kuttl/common/placementapi_deploy.yaml